server.py

```diff
@@ -2,7 +2,7 @@
 # -*- coding: utf-8 -*-
 
 __author__ = "Screenly, Inc"
-__copyright__ = "Copyright 2012-2021, Screenly, Inc"
+__copyright__ = "Copyright 2012-2023, Screenly, Inc"
```

Comment:
Can't really make it 2023 for another 30 days, can we? :)

Reply:
"future-proofing" 😆

Reply:
Can't legally do that I'm afraid.

Reply:
fine, we'll leave this here for 30 days :)